mesonbuild/mesonlib.py

```diff
@@ -430,6 +433,16 @@ def for_darwin(is_cross, env):
     """
     return _get_machine_is_cross(env, is_cross).is_darwin()
 
+def for_linux(is_cross, env):
```

Comment:
__[Flake8]__ [F811] redefinition of unused 'for_linux' from line 416

<sub>posted by Sider</sub>

Reply:
Found it already. Should now be okay. I force-pushed this after a rebase to be as clean as possible for starting in the CI. Hope this is okay.